opensrp-anc/src/test/java/org/smartregister/anc/library/repository/PatientRepositoryTest.java

```diff
@@ -169,6 +169,15 @@ public void testUpdateContactVisitStartDateShouldPassCorrectArgsToUpdateDb() {
         Assert.assertNull(result.get(DBConstantsUtils.KeyUtils.VISIT_START_DATE));
     }
 
+//    @Test
+//    public void isFirstVisitTest() {
+//        String baseEntityId="4faf5afa-fa7f-4d98-b4cd-4ee39c8d1eb1";
+//        PatientRepository patientRepositoryHelper = new PatientRepository();
+//        Assert.assertNotNull(patientRepositoryHelper);
+//        boolean isfistVisit = PatientRepository.isFirstVisit(baseEntityId);
+//        Assert.assertTrue(isfistVisit);
+//    }
+
```

Comment:
Why is this test disabled?

Reply:
I had written this test not knowing PatientRepository isFirstVisit was tested here testSaveAndFinishButtonClickedOpensContactSummarySendPage together with onOptionsSelectedMenuItem()